apps/frontend/src/renderer/components/ScreenshotCapture.tsx

```
@@ -0,0 +1,236 @@
+                  return (
+                    <button
+                      key={source.id}
+                      onClick={() => setSelectedSource(source.id)}
+                      className={cn(
+                        'relative rounded-lg border-2 transition-all overflow-hidden group',
+                        'hover:border-primary/50 hover:shadow-md',
+                        isSelected
+                          ? 'border-primary shadow-lg ring-2 ring-primary/20'
+                          : 'border-border'
+                      )}
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)